test/python/Utils_t/CPMetrics_t.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""
+Unittests for CPMetrics functions
+"""
+
+from __future__ import division, print_function
```

Comment:
Same comment here. We no longer require such compatibility changes.

Reply:
done